sematic/api/endpoints/tests/test_runs.py

```diff
@@ -777,3 +780,23 @@ def test_set_run_user(
         assert payload["content"][0]["user"]["id"] == persisted_user.id
         assert "email" not in payload["content"][0]["user"]
         assert "api_key" not in payload["content"][0]["user"]
+
+
+def test_get_jobs(
```

Comment:
Please add tests for:
- no existent jobs for run id
- no existent run id

Reply:
done